ethereum/contracts/IncentivizedOutboundChannel.sol

```diff
@@ -31,11 +31,13 @@ contract IncentivizedOutboundChannel is OutboundChannel {
         override
     {
         nonce = nonce + 1;
-        emit Message(msg.sender, nonce, payload, msg.value);
+
+
+        emit Message(msg.sender, nonce, payload, gasFee);
```

Comment:
Shouldn't `msg.value` be preserved here?

Reply:
this has nothing to with msg.value. msg.value is for eth. fees are not paid in eth, they're paid in SnowDOT. we should possibly even reject transactions that have any msg.value.

Reply:
Ahh gotcha, my mental model for the Outbound channel was off.